### PR TITLE
Update

### PR DESCRIPTION
**Pull request by redtrillix**
- Edited DESTRONIA_CLA
- Edited CONTRIBUTING.md
- Added more filters to filterlist.
- Sorted filters Alphabetically (A-Z).

Co-Authored-By: JakeWasHere2 <63701873+JakeWasHere2@users.noreply.github.com>

### CODE REVIEW

**Comment by redtrillix** [comment]
@all-contributors please add @redtrillix  for code and tests

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ You will find contribution documentation [here](https://github.com/Destronia/AdB
 
 ## Contributors ✨
 
-Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
+Thanks goes to these wonderful people (code):
```
Comment (JakeWasHere2) [inline README.md:31]: I'd remove the "(code)" section, ive checked the documents and i dont think thats how it works

**Review by JakeWasHere2 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (redtrillix) [reply README.md:31]: > I'd remove the "(code)" section, ive checked the documents and i dont think thats how it works

Oh ok
I dont think its important anyway, ill remove it.

I appreciate the information tho.

this'll do:
```suggestion
Thanks goes to these wonderful people:
```

**Review by JakeWasHere2 — APPROVED** [review]

File: filters/filters.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
-! Version: 6.8.2
+! Version: 6.9.2
```
Comment (JakeWasHere2) [inline filters/filters.txt:1]: Incorrect version number, should be 6.9.0.

Comment (JakeWasHere2) [reply filters/filters.txt:1]:
```suggestion
! Version: 6.9.0
```

Comment (redtrillix) [reply filters/filters.txt:1]: im gonna have to open the pr again.

Comment (redtrillix) [reply filters/filters.txt:1]: nvm ill just use git

**Pull request merged**